### ISSUE TITLE
Better integration with bootstrap style

### ISSUE THREAD

**Issue by sirlordt** [alexgorbatchev/jquery-textext#206]
Hi excellent jquery plugin but... better integration with bootstrap styles needed.

 <div id="divField02DialogEditor" width="98%" sclass="input-group input-group-sm">
     <span id="span01Field02DialogEditor" style="cursor:default;" sclass="input-group-addon z-icon-asterisk" />
    <textarea id="textboxField02DialogEditor" sclass="form-control" rows="1" placeholder="" />

 </div>

The result control look is very ugly.

$( '#textboxField02DialogEditor' ).textext( { plugins : 'arrow autocomplete' } ).bind( 'getSuggestions', function(e, data) { var list = [ 'Basic', 'Closure', 'Cobol', 'Delphi' ], textext = $(e.target).textext()[0], query = (data ? data.query : '') || ''; $(this).trigger( 'setSuggestions', { result : textext.itemManager().filter(list, query) } ); } );

**Comment by Kevinfu510** [comment]
yes, the plugin overrides the bootstrap style, but it works when i manually set the styles to the element.

**Comment by sirlordt** [comment]
i sorry i dont understant the style y set in textarea sclass="form-control" but this no show correct style

**Comment by sirlordt** [comment]
Maybe i missing something?
